### DISCUSSION TITLE
服务端日志info级大量出现 failed to transfer request payload > websocket: close 1000 (normal)

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#159] [Q&A] [answered]
我的配置是vless+tcp+xtls回落使用vless+ws，上网不感到有任何问题，但查服务端日志info级会出现大量这种中断，虽然不影响使用，但总想知道这大概是什么原因？谢谢。
app/proxyman/inbound: connection ends > proxy/vless/inbound: connection ends > proxy/vless/inbound: failed to transfer request payload > websocket: close 1000 (normal)
app/proxyman/inbound: failed to close connection > transport/internet/websocket: failed to close connection > transport/internet/websocket: websocket: close sent
app/proxyman/outbound: failed to process outbound traffic > proxy/freedom: connection ends > context canceled
 app/proxyman/inbound: failed to close connection > tls: use of closed connection

**Comment by C222l** [comment, marked as answer]
我也出现过，大佬告诉我是网络不稳定，我在早上用软银+联通线路也测了下确实也没出现

**Reply by ghost** [reply]
谢谢！
